htdocs/core/modules/import/modules_import.php

```diff
@@ -40,7 +40,12 @@ class ModeleImports
     public $error='';
 
     public $id;           // Id of driver
-	public $label;        // Label of driver
+    
+    /**
+     * @var string proper name for given parameter
```

Comment:
What means "proper name for given parameter" ?

Reply:
it's a standard comment before finding the write one...

Reply:
^the right one...